server/routes/v1/deployment.ts

```diff
@@ -161,3 +165,59 @@ export const resetDeploymentApprovals = [
     return res.json(deployment)
   },
 ]
+
+export const fetchRawModelFiles = [
+  ensureUserRole('user'),
+  bodyParser.json(),
+  async (req: Request, res: Response) => {
+    const { uuid, version, fileType } = req.params
+    const deployment: any = await findDeploymentByUuid(req.user!, uuid)
+
+    if (!req.user._id.equals(deployment.owner)) {
+      throw Unauthorised(
```

Comment:
You should include things like the deployment owner in the first argument here.

Reply:
added